### DISCUSSION TITLE
Welcome to ner-annotator Discussions!

### DISCUSSION THREAD

**Discussion by tecoholic** [tecoholic/ner-annotator#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jyotirmayg** [comment]
I would try to install this ner annotation tool on my system but it is coming up with error while installation.
![Screenshot 2022-01-18 112125](https://user-images.githubusercontent.com/67102212/149878682-1348f850-ebf8-48b8-b4a4-f0820b4f758a.png)


Kindly help me to fix this bug.

**Reply by tecoholic** [reply]
@jyotirmayg I am not a windows user. But from the error message, it looks like you need Visual Studio to be installed.

**Reply by jyotirmayg** [reply]
Thanks for your reply. I already tried re-installing the visual studio but it didn't work for me. I will try installing it in the Linux system.

**Comment by AswiN-7** [comment]
is this ner annotator will not work in windows?

**Reply by tecoholic** [reply]
This can be used directly from the browser. Just go to https://tecoholic.github.io/ner-annotator/

**Reply by AswiN-7** [reply]
our data is very large so it will be help full if annotator run locally

**Reply by tecoholic** [reply]
The web version doesn't upload content anywhere, it just loads the file onto the memory just like the desktop version. For large datasets, it is advisable to split the data into multiple files and then annotate. Otherwise you will face issues both in web as well as desktop version.

For desktop only Linux is available.

**Reply by AswiN-7** [reply]
thank you anna

**Reply by tecoholic** [reply]
@AswiN-7 Now you can install NER Annotator in Windows https://github.com/tecoholic/ner-annotator/releases/download/v1.0.2/NER.Annotator_1.0.2_x64.msi

**Reply by jyotirmayg** [reply]
@tecoholic Thank you very much. that's helpful.

**Reply by AswiN-7** [reply]
thank you anna

**Comment by SvM229** [comment]
Hi, this annotation tools gives us a json file in the end. How do I use this in transformers library by huggingface, or is there any way that it outputs the data in that format?

**Reply by tecoholic** [reply]
I am not aware of the Huggingface transformers. This tool was created to help create annotations for spacy. See the [spaCy documentation](https://spacy.io/api/data-formats#json-input) about converting this JSON into appropriate input for spaCy 3.0 and above.

**Reply by SvM229** [reply]
Thanks for the reply.

**Comment by PriyeshDave** [comment]
Hi,
This application looks perfect for labelling task. I have a question regarding the security. What if we use the web application, will my data get stored? Will it be safe?

**Reply by tecoholic** [reply]
The data never leaves your computer. Everything happens in memory and the tags are stored in your browser's localstorage.
